oat-sa/tao-item-runner-qti-fe
# feature: TR-892 apply correct layout for RTL items in RTL tests

**Related to:** https://oat-sa.atlassian.net/browse/TR-892

**Changes:**
- support RTL items in LTR tests

**How to check:**
- checkout necessary branch of `taoQtiItem`
- go to views dir and run `npm install` 
- register any language to check as RTL in `config/tao/client_lib_config_registry.conf.php` (en-GB  as example)
```
    'config' => array(
        'util/locale' => array(
            'decimalSeparator' => '.',
            'thousandsSeparator' => '',
            'rtl' => array(
                'ar-arb',
                'en-GB'
            )
        ),
```
- import test with pre-defined RTL language for any item
- walk through imported test and ensure it rendered as RTL
